src/components/views/audio_messages/SeekBar.tsx

```diff
@@ -60,7 +60,7 @@ export default class SeekBar extends React.PureComponent<IProps, IState> {
         super(props);
 
         this.state = {
-            percentage: 0,
+            percentage: percentageOf(this.props.playback.timeSeconds, 0, this.props.playback.durationSeconds),
```

Comment:
does this affect voice messages or audio messages?

Reply:
It is used there. Tested them, they are working as expected.